R/readWriteMgfData.R

```diff
@@ -45,13 +60,17 @@ writeMgfDataFile <- function(splist, con, COM = NULL, TITLE = NULL,
     if (verbose)
       setTxtProgressBar(pb, i)
 
-    writeMgfContent(splist[[i]], TITLE = NULL, con = con)
+    if (haveAddFields)
+        writeMgfContent(splist[[i]], TITLE = NULL, con = con,
+                        addFields = addFields[i, ])
+    else
+        writeMgfContent(splist[[i]], TITLE = NULL, con = con)
   }
   if (verbose)
     close(pb)
 }
 
-writeMgfContent <- function(sp, TITLE = NULL, con) {
+writeMgfContent <- function(sp, TITLE = NULL, con, addFields = TRUE) {
```

Comment:
I don't understand the default value `TRUE` for `addFields` here.

Reply:
Ups, well, me neither. I'll change to `addFields = NULL`.